Wireframe/css/style.css

```diff
@@ -0,0 +1,309 @@
+
+
+/* Basic settings */
+body {
+    margin: 0;
+    font-family: Arial, sans-serif;
+    background-color: #f4f4f4;
+    color: #333;
+}
+
+/* Header */
+header {
+    text-align: center;
+    padding: 25px;
+    padding-top: 2vw;
+    background-color: #0c0f0a;
+    color: #fff;
+    border: 3px solid #f5cb5c;
+}
+
+header h1 {
+    margin: 0;
+    font-size: 2.5rem;
+    color: #f5cb5c;
+}
+
+header p {
+    font-size: 1.3rem;
+    color: #ccc;
+}
+
+/* Main Layout */
+main {
+    display: block;
+    padding: 0px;
+    background-color:#333533 ;
+    
+}
+
+/* Main article */
+.main-article {
+    background-color: #333533;
+    color: #f4f4f4;
+    padding: 0;
+    width: 100vw;
+    max-width: 1514px;
+    margin: 0 auto ; 
+    box-shadow: 0 4px 8px rgba(0, 0, 0, 0.1);
+    border: 3px solid #f5cb5c;
+}
+
+.main-article img {
+    width: 100%;  
+    height: 420px;
+    
+    
+}
+
+.main-article h2 {
+    margin-top: 15px;
+    color: #f5cb5c;
+    font-weight: bold;
+    padding: 0 10px;
+    
+}
+
+.main-article p {
+    font-size: 18px;
+    line-height: 1.6;
+    padding: 0 10px 10px;
+}
+
+/* Sub-articles container */
+.sub-articles {
+    display: flex;
+    flex-direction: row;
+    max-width: 100%;
+    margin: 0 ;
+    /* gap: 20px; */
+    margin-bottom: 60px;
+    
+}
+
+
+
+/* Sub-article styling */
+.sub-article {
+    display: flex;
+    flex-direction: column;
+    background-color: #333533;
+    flex: 1;
+    padding: 0px;  /* Sub-article styling */
+    box-shadow: 0 4px 8px rgba(0, 0, 0, 0.1);
+    margin: 0;
+    border: 4px solid #f5cb5c;
+    
+    
+}
+    /* first child */
+.sub-article:first-child {
+    margin-top: 0;
```

Comment:
How would setting this property make the "first child" any different from other elements of class `.sub-article`?
Does this selector selects the first article in the webpage?

Reply:
Hello ciyuan, and thank you very much for reviewing my work. Below, you'll find the updates I've made. I look forward to any further feedback. Thank you so much!